services/web-app/components/markdown/markdown.module.scss

```diff
@@ -17,8 +17,9 @@
   margin-top: spacing.$spacing-06;
 }
 
+// override styles on component below h4 to not have any spacing
 .h4-container + * {
-  margin-top: 0;
+  margin-top: 0 !important;
```

Comment:
any chance we can use a more specific class to avoid using important? if not I'm good with it 👍

Reply:
fixed